src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,34 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int ARRAY_LENGTH = 12;
+    public static final int EMPTY = -1;
+    private final int[] numbers = new int[ARRAY_LENGTH];
 
-    public int total = -1;
+    public int total = EMPTY;
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean callCheck() {
-        return total == -1;
+        return total == EMPTY;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == ARRAY_LENGTH - 1;
     }
 
     protected int peekaboo() {
         if (callCheck())
-            return -1;
+            return EMPTY;
         return numbers[total];
     }
 
     public int countOut() {
         if (callCheck())
-            return -1;
+            return EMPTY;
         return numbers[total--];
     }
```

Comment:
I don't think you should use `EMPTY` here as this is more like returned error code than empty array length.
In the other file you return `EMPTY_RETURN_VALUE` on empty and I think it would be a better name.